### PR TITLE
add japanese translate

### PR DESCRIPTION
**Pull request by Jxck**
Hi all.

I wanna advocate this manifesto and principle of Extensible Web to Japanese web devs.

I think this thought is quite important for web future, so I wanna join this as possible as I can.
in japan, there are lots of powerful web developer, but it seem for me that they don't no about this manifesto well.
at first, I translate this in Japanese. but I don't know how publish it.

so I create this PR for asking you.

if it's works fine for you, I'm appreciate it if you publish this for example at https://ja.extensiblewebmanifesto.org/ 
or if that dosen't works. please let me to publish this at my own domain (https://jxck.io) or on my blog (http://jxck.hatenablog.com).

also, thanks for your every works for web.

thanks, and sorry for my poor english ;p

Jxck

### CODE REVIEW

**Comment by yoheiMune** [comment]
This is pretty good, I feel so, and I also definitely want to expand this idea to Japanese developers.
I hope this request will be accepted.

**Comment by harry0000** [comment]
Great job! :+1: 

I have verified this translation , so I found some orthographical variants.

- 「元に」 <-> 「もとに」
- 「回る」 <-> 「まわします」
- 「既に」 <-> 「すでに」
- 「Web開発者」 <-> 「Web の開発者」

**Comment by Jxck** [comment]
updated, thanks @harry0000 :)

**Comment by agektmr** [comment]
LGTM

**Comment by bkardell** [comment]
Thanks everybody, I have one more person who said they'd look it over in the next few hours, plan to merge barring anything unforseen when I wake up US EST

**Comment by kinu** [comment]
This looks good to me too, and it'd be great to have translations on the site!

**Comment by kinu** [comment]
One nit comment:

- 「Mozilla's X-Tags や Google's Polymer」 → 「Mozilla の X-Tags や Google の Polymer」would be probably more natural

**Comment by h13i32maru** [comment]
LGTM!
[IMO] "新しい特徴を説明することは" -> "新しい機能を説明すると、"

**Comment by Jxck** [comment]
updated thanks @kinu.

**Comment by Jxck** [comment]
thanks @h13i32maru 
but I think it's better as-is. "by explaining ~" it to be "説明することは〜" or "説明することで〜".

**Comment by Jxck** [comment]
thanks @bkardell  and everyone ;)

File: README.ja.md [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+第一の目標は、Web 標準仕様のエディタと Web 開発者の間で、しっかりとフィードバックのループがまわることです。
+標準化が発展していくモデルとして、手順を重視しトップダウンで決定するモデルよりも、多くの有能な開発者によって駆動するモデルを選びます。
+
+ブラウザベンダは、基盤となるプラットフォームが持つ可能性を、低レベルの機能としてできる限り提供すべきです。
+新しい高レベル API を JavaScript による実装をもとに議論できるように、土台を提供すべきです(すでに Mozilla の [X-Tags](http://www.x-tags.org/) や Google の [Polymer](http://www.polymer-project.org/) で行われています)。
+これにより、開発者は標準化に参加可能になり、ブラウザの外で議論が繰り返されることで、粗悪な API が定義されるのを防ぎます。
+
+特に、 Extensible Web プラットフォームに対して、以下の方針を提唱します。
+* 標準化プロセスは、 Web プラットフォームに対するセキュアで効果的な ***新しい低レベルな機能*** の策定に注力すべきです。
+* Web プラットフォームは、開発者が HTML や CSS などを理解し、それらを置き換えることが可能となる ***既存の特徴を説明*** した低レベルな機能を提供すべきです。
```
Comment (kwkbtr) [inline README.ja.md]: The meaning of the sentence seems to be slightly different from that of the english one.
I propose:

「Web プラットフォームは、 HTML や CSS などの ***既存の特徴を説明*** する低レベルな機能を提供することで、開発者がそれを理解し再現することが可能となるようにすべきです。」

Comment (Jxck) [inline README.ja.md]: this trans seems fine for me.
I'll fix it, thanks !

**Comment by yuskesh** [comment]
I prefer "編集者" than "エディタ". "エディタ" sounds like editor software.

And I've been wondering "低レベルな機能" is equivalent of "low-level capabilities" here. I think some meanings are lost in translation.

Sorry, I don't come up with proper translation for now.

**Comment by Jxck** [comment]
but I don't know word exactly same meaning with "low-level capabilities".
we never say "低レベルな能力" in usual tech talk.
"機能" used multiple mean including 'feature' and 'capabilities' in Japan,
and ja <-> en is not 1:1.

**Comment by Jxck** [comment]
I think we use 'エディタ' in standardization context.
and usually 'Editor' is '編集者' in Japanese, but 'spec editor' actually means like '執筆者', '策定者' etc.
so I think 'エディタ' seems fit with in this context, rather than force Japanese un-much word.

anyway, thanks for review ysukesh :)

**Comment by yuskesh** [comment]
About 'エディタ', then I'd suggest to use 'エディター'. It's super tiny problem though.
Omitting 'ー' is only used in technical term.
see http://ja.wikipedia.org/wiki/%E9%95%B7%E9%9F%B3%E7%AC%A6#.E9.95.B7.E9.9F.B3.E7.AC.A6.E3.81.AE.E7.9C.81.E7.95.A5

**Pull request merged**